File: 5-interprete/tests/micro11.py
def main() -> None:
	numero = 0
	x =0
	print("Digite um numero: ")
	int(input(numero))
	numero = verifica(numero)

	if numero == 1:
		print("Positivo")
	elif numero == 0:
		print("zero")
	else:
		print("Negativo")


def verifica(n:int) -> int:
	res = 0
	if n > 0:
		return 1
	if n < 0:
		return 3
	else:
		return 0

main()

File: 1-codes/micro/micro11.c
#import<stdio.h>

int verifica(int n) {
  int res;
  if (n > 0) {
    res = 1;
  } else if (n < 0) {
    res = -1;
  } else {
    res = 0;
  }
  return res;
}

int main() {
  int numero, x;

  printf("Digite um numero: ");
  scanf("%d", &numero);

  x = verifica(numero);
  if(x == 1) {
    printf("Numero Positivo");
  } else if(x == 0) {
    printf("Zero");
  } else {
    printf("Negativo");
  }

  return 0;
}
